### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by smilingthax**
Issue #2

### CODE REVIEW

File: lib/macho/parser.js [hunk 1]
```diff
@@ -389,3 +391,42 @@ Parser.prototype.parseLinkEdit = function parseLinkEdit(type, buf) {
...
+Parser.prototype.parseFunctionStarts = function parseFunctionStarts(type, buf, file) {
+  if (buf.length !== 8)
+    throw new Error('function_starts OOB');
+
+  var dataoff = this.readUInt32(buf, 0);
+  var datasize = this.readUInt32(buf, 4);
+  var data = file.slice(dataoff, dataoff + datasize);
+
+  var addresses = [];
+  var address = 0; // TODO? start address / "base address" is vmaddr of the first (non-null [e.g. initprot!=0]) segment (i.e. __TEXT ?)
```
Comment (indutny) [inline lib/macho/parser.js]: 80 column limit, please.

**Comment by indutny** [comment]
Looking really good, thank you! (and sorry for delay!)

Do you think that a test case could also be contributed along with this change?

Thanks again.

**Comment by trufae** [comment]
what's missing to merge this?

**Comment by indutny** [comment]
Nothing.

**Comment by indutny** [comment]
Thank you!

**Pull request merged**